pisek/jobs/parts/task_job.py

```diff
@@ -183,3 +190,25 @@ def _diff_files(self, file_a: TaskPath, file_b: TaskPath) -> str:
             stdout=subprocess.PIPE,
         )
         return diff.stdout.decode("utf-8")
+
+    def _file_content(
+        self,
+        file: TaskPath,
+        force: bool = False,
+        tab_times: int = 1,
```

Comment:
What is the intended usecase for this? The convention seems to be that indenting text is the responsibility of whoever combines it with other text. I don't think this function should indent the result at all, that should be left to `_named_file`.

Reply:
This gets very annoying if we are actualy not printing anything when ``env.file_contents == False``.

Reply:
What do you mean? It's true that you would have to add an `if` to `_named_file`, but I don't see how that's more annoying than adding a for loop here. I'm also unsure if this function shold be responsible for checking the `env`. Shouldn't that also be done in `_named_file`?